README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# MACS 20500: Homework 1  
+## James Zimmerman  
+
+  
+### Intro  
+Hi, I'm James Zimmerman! I'm a rising third year here at UChicago majoring in **Public Policy** and in **Economics**.
+
+### Facts About Me  
+  + I'm from **New York City**, living my entire life in the same apartment before leaving for college.  
+  + I've become really interested in studying **film** over my second year. I can enjoy any movie that isn't horror, no matter how good or bad.  
+  + I also enjoy **history**, particularly Reconstruction and the 20th-century United States.  
+  + I played basketball and baseball growing up. Today, I too closely follow the **NBA** and the Pittsburgh Steelers.  
+
+### Reflections  
```

Comment:
In the future, add your reflections to your **pull request comment** rather than within the assignment itself. That way your graders/reviewers can see what you enjoyed and struggled with directly in the pull request page.